src/neo/VM/Helper.cs

```diff
@@ -349,6 +351,9 @@ private static StackItem ToStackItem(ContractParameter parameter, List<(StackIte
                     stackItem = (string)parameter.Value;
                     break;
                 case ContractParameterType.InteropInterface:
+                case ContractParameterType.Any:
```

Comment:
`Any` was thought for returning different kind of types, is like an object, so it's not the same as `null`. I understand the problem but this could produce other's I think.

Reply:
But the `Deserialize` method just pushes StackItem.Null when meeting `Any`. And `InteropInterface` is more like an object.

![image](https://user-images.githubusercontent.com/4393234/72774307-128f1b80-3c45-11ea-9eec-8464fd3cc5fb.png)

Reply:
I think that this should throw an error, because it's not possible to know which value is

Reply:
> But the Deserialize method just pushes StackItem.Null when meeting Any. And InteropInterface is more like an object.

They are different. `Deserialize()` uses the data from `Serialize()` only.

Reply:
@chenquanyu could you remove this case?

Reply:
So we just throw an exception when meeting `ContractParameterType.Any`?

Reply:
Yes, I think that it's better

Reply:
OK, I will change it. How about also change `ToParameter` to make `StackItem.NULL` as `ContractParameterType.Void`?

Reply:
I think that we are mixing method descriptions with Parameters, `void` it's not the same as null, maybe it's better to use two enums, one for parameters and other for `returns`

Reply:
It seems that the `ContractParameterType.Void` is not used in neo projects, maybe we can remove it. Then `ToParameter` and `ToStackItem` methods are just for JSON converting, maybe we can add `ContractParameterType.Null` to match `StackItem.Null`😂

Reply:
I prefer create `ContractReturnType` and move some entries from `ContractParameterType`

Reply:
I think there is no need to create `ContractReturnType`. `Void` can not be used as a parameter.